scripts/install/default-env-vars.sh

```diff
@@ -5,4 +5,4 @@ export PANLOG=${PANDIR}/logs
 export POCS=${PANDIR}/POCS
 export PAWS=${PANDIR}/PAWS
 export PIAA=${PANDIR}/PIAA
-export MPLBACKEND=${POCS}/.matplotlib
\ No newline at end of file
+export MATPLOTLIBRC=${POCS}/.matplotlib
```

Comment:
There is no such file in POCS. Do you have one?

Do you want to rename `POCS/examples/notebooks/matplotlibrc` to `POCS/matplotlibrc` (not `POCS/.matplotlib`)?

Reply:
The env var can point to a file or a directory and that directory should now exist: https://github.com/panoptes/POCS/tree/develop/.matplotlib

Reply:
My apologies, I was looking for a file in the github UI, and not a directory, which are grouped separately from files. Sigh.

Reply:
We should perhaps set this env var in conftest.py (in a future PR). I'd like to have fewer rather than more env vars.